### PR TITLE
Modify the info passed to ScaleDownStatusProcessor when empty nodes a…

### PR DESCRIPTION
**Pull request by towca**
…re deleted

Previously, if any of the nodes fails to delete, the processor gets
a ScaleDownError status. After this commit, it will get the list of
nodes that were successfully deleted.

### CODE REVIEW

File: cluster-autoscaler/core/scale_down.go [hunk 1]
```diff
@@ -911,32 +922,34 @@ func (sd *ScaleDown) scheduleDeleteEmptyNodes(emptyNodes []*apiv1.Node, cp cloud
 					metrics.RegisterScaleDown(1, gpu.GetGpuTypeForMetrics(cp.GPULabel(), cp.GetAvailableGPUTypes(), nodeToDelete, nodeGroup), metrics.Unready)
 				}
 			}
-			confirmation <- deleteErr
+			confirmation <- emptyNodeDeletionConfirmation{node: node, err: deleteErr}
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/core/scale_down.go]: Let's drop the 'empty' part from emptyNodeDeleteConfirmation (both the type and variables). It's sent for both empty and non-empty nodes.

Comment (towca) [reply cluster-autoscaler/core/scale_down.go]: In my understanding, this method and this channel is only used for empty nodes, but no problem!

Comment (aleksandra-malinowska) [reply cluster-autoscaler/core/scale_down.go]: Right, nevermind:/ Shorter name is still nice, but that's just a nit in this case.

Comment (towca) [reply cluster-autoscaler/core/scale_down.go]: Done already :)

**Pull request merged**